src/com/cft/contactmerge/LastNameMatcher.java

```diff
@@ -0,0 +1,34 @@
+/* Created by James Page on May 8th, 2018
+ * Contains the order in which the last names will be considered a match or not...
+ * Exact Match (and matched with punctuations), matched with hyphenated parts, matched with apostrophe.
+ */
+
+package com.cft.contactmerge;
+
+import java.util.ArrayList;
+
+public abstract class LastNameMatcher extends NameMatchLogic {
```

Comment:
Similar comment as above... "extends NameMatchLogic" is confusing because it suggests a relationship between these classes that doesn't currently exist.